src/main/java/org/opensearch/knn/index/SpaceType.java

```diff
@@ -170,4 +175,14 @@ public static SpaceType getSpace(String spaceTypeName) {
         }
         throw new IllegalArgumentException("Unable to find space: " + spaceTypeName);
     }
+
+    /**
+     * Translate a score to a distance for this space type
+     *
+     * @param score score to translate
+     * @return translated distance
+     */
+    public float scoreToDistanceTranslation(float score) {
```

Comment:
Why not make abstract and implement for all of the space types?

Reply:
Because for other space types, currently I don't see the use cases for engines as common, we can change to abstract and add implement whenever needed.